mitxgraders/helpers/validatorfuncs.py

```diff
@@ -83,6 +84,49 @@ def is_callable(obj):
     """Returns true if obj is callable"""
     return callable(obj)
 
+def is_callable_with_args(num_args):
+    """
+    Validates that a function is callable and takes num_args arguments
+
+    Examples:
+    >>> def func(x, y): return x + y
+    >>> is_callable_with_args(2)(func) == func
+    True
+    >>> is_callable_with_args(3)(func) == func # doctest: +ELLIPSIS
+    Traceback (most recent call last):
+    Invalid: Expected function... to have 3 arguments, instead it has 2
+
+    Callable objects work, too:
+    >>> class Foo:
+    ...     def __call__(self, x):
+    ...         return x
+    >>> foo = Foo()
+    >>> is_callable_with_args(1)(foo) == foo
+    True
+    >>> is_callable_with_args(1)(Foo) # doctest: +ELLIPSIS
+    Traceback (most recent call last):
+    Invalid: Expected function... to have 1 arguments, instead it has 2
+    """
+    def _validate(func):
+        # first, check that the function is callable
+        is_callable(func)
```

Comment:
Does this raise an error if it isn't callable? Might be worth adding a comment if so.